### DISCUSSION TITLE
Selecting Energy groups in Gromacs+Martini

### DISCUSSION THREAD

**Discussion by ajhaver4** [Martini-Force-Field-Initiative/MartiniDiscussions#6] [Proteins]
Hi All,
I have a protein + water system and I want to recalculate the energies w.r.t certain water molecules. I am using the gromacs selection tool to generate a new index file with the specified multiple group of water molecules. After generating a new .tpr files, I then use the gmx mdrun command with the -rerun option to recalculate the energies. 

I am facing a peculiar issue when I output the energies (by using the gmx energy command). The LJ-SR interactions only show up in one group of the water molecules and show a value of 0 for all other groups of water molecules. After further troubleshooting, I found that the LJ-SR interaction is only non-zero for the group which includes the 1st water molecule (after the protein or ion molecules). And this value is equal to the total sum of LJ-SR interactions of all "WN" molecules. 

It seems puzzling to me why the pairwise LJ-SR interactions appears only in a particular group and not in any other water groups. Am I missing something with how Martini calculates the energies or with how gromacs interprets the indexed groups? 

I'm happy to forward the index and trajectory files

**Comment by amiel-ab** [comment]
Hi @ajhaver4 
I guess you already  have specified different groups and check that each bead is present in the correct group in the index file.
Have you specified  the "energy_grps" option in the .mdp as well before using rerun ?  
Amiel
